docker/nginx/nginx.conf

```diff
@@ -0,0 +1,78 @@
+user                    www-data www-data;
+worker_processes        5;
+error_log               /etc/nginx/logs/error.log;
+pid                     /etc/nginx/nginx.pid;
+worker_rlimit_nofile    8192;
+
+events {
+  worker_connections  4096;
+}
+
+http {
+    include    /etc/nginx/mime.types;
+    include    /etc/nginx/proxy.conf;
+    include    /etc/nginx/fastcgi.conf;
+
+    default_type application/octet-stream;
+    server_names_hash_bucket_size 128;
+    log_format scripts '$document_root$fastcgi_script_name > $request';
+
+    sendfile            on;
+    sendfile_max_chunk  1m;
+    tcp_nopush          on;
+    tcp_nodelay         on;
+    keepalive_timeout   65;
+
+    gzip on;
+    gzip_min_length 1000;
+    gzip_buffers 4 8k;
+    gzip_types text/plain application/x-javascript text/css image/png image/jpeg image/gif image/x-icon image/svg+xml;
+    gzip_vary on;
+
+    add_header X-XSS-Protection         "1; mode=block";
+    add_header X-Content-Type-Options   nosniff;
+    add_header Content-Security-Policy  "default-src 'self' https://www.googleapis.com; script-src 'self' 'unsafe-inline'; img-src 'self' data:; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/; font-src 'self' https://fonts.gstatic.com/; frame-src 'self'; object-src 'none'";
+
+    proxy_cache_path        /etc/nginx/cache levels=1:2 keys_zone=web_cache:1m max_size=256m inactive=60m use_temp_path=off;
```

Comment:
What exactly is being cached here? As far as I can tell you don't have any `proxy_pass` directives anywhere, so this won't do anything.
Also I don't think there is anything you want to cache because nginx is only serving static files and responses from php-fpm (which you definitely don't want to cache in this case).

Reply:
Ah yes... I should've used `fastcgi_cache`. I can imagine a scenario in which (in the future) some responses may be cached. Not sure if a future version of Zend supports that though.

Reply:
If someone is logged in to the website you don't want the response to get cached, because then you might end up returning that to another user.

Reply:
That's true. nginx determines what to cache and what not to cache based on the cache-related headers according to the documentation. Since all requests from the website currently return with something like this, we shouldn't worry about personal details getting cached.
![afbeelding](https://user-images.githubusercontent.com/32361020/114308558-aff19c00-9ae4-11eb-8df1-84801c49dd4d.png)